File: adapters/laravel-inertia/routes/health.php
<?php

use Illuminate\Support\Facades\Route;

// Loaded by bootstrap/app.php's withRouting(then: ...); laravel loads only
// web.php and api.php itself. The database driver splices in the probe; with
// --db none the throw stands and readiness reports 503.
Route::get('/health/ready', function () {
    try {
        // @DB_PROBE@
        throw new RuntimeException('no database is configured for this project');
    } catch (Throwable $e) {
        // Logged, not returned: the error names host and user, and this route is
        // unauthenticated. logger(), not Log: another `use` breaks pint's import
        // order once the driver splices in DB.
        logger()->warning('readiness probe failed: '.$e->getMessage());

        return response()->json(['status' => 'unavailable'], 503);
    }
});

File: adapters/flask/app/health.py
from flask import Blueprint, Response, current_app, jsonify

# @DB_ENGINE@

health = Blueprint("health", __name__)

Reply = Response | tuple[Response, int]


@health.get("/health/live")
def live() -> Reply:
    return jsonify(status="ok")


@health.get("/health/ready")
def ready() -> Reply:
    try:
        # @DB_PROBE@
        raise RuntimeError("no database is configured for this project")
    except Exception as error:  # noqa: BLE001
        # Logged, not returned: the error names host and user, and this route is
        # unauthenticated.
        current_app.logger.warning("readiness probe failed: %s", error)
        return jsonify(status="unavailable"), 503
